Branch-SDK/Branch-SDK/BranchGetAppListRequest.h

```diff
@@ -1,16 +0,0 @@
-//
```

Comment:
Right, the problem was that they were failing to be parsed by the `retrieve` function, so now there's a try/catch wrapped around that, which ensures that if the request is of an unknown type, it doesn't get added to the queue (and doesn't cause the app to crash). Try it out locally -- run on master, get the request enqueued, then switch to this branch -- should work :smiley: